js/react/components/home/PartnerCarousel.jsx

```diff
@@ -0,0 +1,51 @@
+import { Carousel } from '@mantine/carousel';
+import { Image, MantineProvider } from '@mantine/core';
+import Autoplay from 'embla-carousel-autoplay';
+import { useRef } from 'react';
+
+import { PARTNER_CAROUSEL_DATA } from '../../utils/constants';
+
+function PartnerCarousel() {
```

Comment:
you can do the export in this line, there are no proptypes applied on the component